src/test/kotlin/pl/edu/agh/gem/internal/service/UserDetailsServiceTest.kt

```diff
@@ -95,4 +97,56 @@ class UserDetailsServiceTest : ShouldSpec({
         }
         verify(userDetailsRepository, times(1)).findById(USER_ID)
     }
+
+    should("update UserDetails successfully") {
+        // given
+        val oldUserName = "name1"
+        val newUserName = "name2"
+        val oldAttachmentId = "123"
+        val newFirstName = "John"
+        val newLastName = "Doe"
+
+        val userDetails = createBasicUserDetails(USER_ID, oldUserName, oldAttachmentId)
+        val userDetailsUpdate = createUserDetailsUpdate(
+            USER_ID,
+            newUserName,
+            newFirstName,
+            newLastName,
```

Comment:
could you use in all places style like `id = USER_ID, username = newUserName` etc

Reply:
done